README.md

```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
 pip install merlin-systems
 ```
 
+### Installing Merlin Systems from Source
+
+Merlin Systems can be installed from source by cloning the GitHub repository and running `setup.py`
+
+```bash
+git clone https://github.com/NVIDIA-Merlin/systems.git
+cd systems && python setup.py develop
+```
+
+### Running Merlin Systems from Docker
```

Comment:
I think we might want to keep the table of which libraries are installed on which containers at the Merlin level